packages/js/plugins/http/src/index.ts

```diff
@@ -13,6 +14,9 @@ import {
   PluginPackage,
 } from "@web3api/core-js";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
+const formData = require("form-data");
```

Comment:
Can we use a types package for the form-data package? That way we don't have to do `require(...)`?